storybook/src/elements/nuxeo-document-layout/nuxeo-document-layout.stories.js

```diff
@@ -0,0 +1,174 @@
+import { storiesOf } from '@storybook/polymer';
+import { html } from 'lit-html';
+import { until } from 'lit-html/directives/until.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import { select } from '@storybook/addon-knobs';
+import { LayoutBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-layout-behavior.js';
+import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-document-layout.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-input.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date-picker.js';
+import hljs from 'highlight.js';
+import '../../../.storybook/i18n';
+import image from '../../img/image01.jpg';
+import DocumentBuilder from '../../data/documents.data';
+/* import '../code-highlighter.js'; */
+
+window.Polymer = Polymer;
+window.Nuxeo.LayoutBehavior = LayoutBehavior;
+
+const documentBuilder = new DocumentBuilder().setTitle('My Document').setFileContent('Nuxeo Logo', image);
+
+const codePanel = (type, layout) => html`
+  <style>
+    .hljs {
+      display: block;
+      overflow-x: auto;
+      padding: 0.5em;
+      color: #abb2bf;
+      background: #282c34;
+      margin: 0;
+    }
+
+    .hljs-comment,
+    .hljs-quote {
+      color: #5c6370;
+      font-style: italic;
+    }
+
+    .hljs-doctag,
+    .hljs-keyword,
+    .hljs-formula {
+      color: #c678dd;
+    }
+
+    .hljs-section,
+    .hljs-name,
+    .hljs-selector-tag,
+    .hljs-deletion,
+    .hljs-subst {
+      color: #e06c75;
+    }
+
+    .hljs-literal {
+      color: #56b6c2;
+    }
+
+    .hljs-string,
+    .hljs-regexp,
+    .hljs-addition,
+    .hljs-attribute,
+    .hljs-meta-string {
+      color: #98c379;
+    }
+
+    .hljs-built_in,
+    .hljs-class .hljs-title {
+      color: #e6c07b;
+    }
+
+    .hljs-attr,
+    .hljs-variable,
+    .hljs-template-variable,
+    .hljs-type,
+    .hljs-selector-class,
+    .hljs-selector-attr,
+    .hljs-selector-pseudo,
+    .hljs-number {
+      color: #d19a66;
+    }
+
+    .hljs-symbol,
+    .hljs-bullet,
+    .hljs-link,
+    .hljs-meta,
+    .hljs-selector-id,
+    .hljs-title {
+      color: #61aeee;
+    }
+
+    .hljs-emphasis {
+      font-style: italic;
+    }
+
+    .hljs-strong {
+      font-weight: bold;
+    }
+
+    .hljs-link {
+      text-decoration: underline;
+    }
+  </style>
+  <div style="margin-top: 32px">
+    <span style="margin-left: 4px; color: #ddd">layout source</span>
+    <span class="hljs">
+      ${until(
+        import(`!!raw-loader!../../../public/layouts/document/${type}/nuxeo-${type}-${layout}-layout.html`).then(
```

Comment:
thought you had set up to use the raw loader for HTML files, no?

Reply:
IIRC, html files are being processes by the `polymer-webpack-loader`, a peer dependency of `@storybook/polymer` (see https://github.com/storybookjs/storybook/blob/master/app/polymer/package.json#L51)

Reply:
I thought the changes done in .storybook/webpack.config.js to use the `raw-loader` for `layout.html` was for these use cases, no?

Reply:
`polymer-webpack-loader` still interferes and we see a bunch of warnings on the console, even if it seems to work.